web/modules/custom/bnf/bnf_example_content/bnf_example_content.module

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Logger\LoggerChannelFactoryInterface;
+use Drupal\drupal_typed\DrupalTyped;
+
+/**
+ * Implements hook_modules_installed().
+ *
+ * @param string[] $modules
+ *   The names of the modules that were installed.
+ */
+function bnf_example_content_modules_installed(array $modules): void {
+  // Only react to our own installation.
+  if (!in_array('bnf_example_content', $modules)) {
+    return;
+  }
+
+  $logger = DrupalTyped::service(LoggerChannelFactoryInterface::class, LoggerChannelFactoryInterface::class)->get('bnf_example_content');
```

Comment:
Consider if `DrupalTyped` is necessary here.

When retrieving loggers I think `\Drupal::logger()` is fine.